ts/components/NavBarLabel.tsx

```diff
@@ -32,6 +32,13 @@ const ROUTE_LABEL: RouteLabelMap = {
 };
 const fallbackLabel = "unknown"; // fallback label
 
+const routeOrder = new Map([
```

Comment:
```suggestion
const routeOrder = new Map<Routes, number>([
```
To avoid typos or misleading strings you could specify the type allowed as key

Reply:
The problem is that we use as key to read from the Map the `routeName` in the props that is a string
```typescript
type OwnProps = {
  options: {
    tintColor: string | null;
    focused: boolean;
  };
  routeName: string;
};
```
and it comes from `NavigationScreenProp<NavigationRoute>` (from react-navigation) that is a string too. We cannot cast a string to a `Routes`.